addons/im_livechat/static/src/components/composer/composer.xml

```diff
@@ -5,6 +5,9 @@
         <xpath expr="//*[hasclass('o_Composer_buttonAttachment')]" position="replace">
             <t t-if="!composer.thread or composer.thread.channel_type !== 'livechat'">$0</t>
         </xpath>
+        <xpath expr="//t[@t-if='isDropZoneVisible.value']" position="replace">
```

Comment:
This is working for 14.0 and 15.0, but will not work in master.
But I don't think we want to rely on an xpath for this feature, this will break if for some reason we change the `t-if` condition. 
Since your condition is only on `channel_type` I would instead create a `isDropZoneDisabled` computed field on the `composer` model in 14.0, `composerView` model in 15.0 and finally on `dropZoneView` model in master.